src/gaggiuino.ino

```diff
@@ -800,7 +800,7 @@ static inline void sysHealthCheck(float pressureThreshold) {
 
   /* This *while* is here to prevent situations where the system failed to get a temp reading and temp reads as 0 or -7(cause of the offset)
```

Comment:
It seems that the part of the comment "or -7(cause of the offset)" is not vaid anymore.